tests/foreman/ui/test_discoveredhosts.py

```diff
@@ -14,18 +14,18 @@ class Discovery(UITestCase):
     """Implements Foreman discovery tests in UI."""
 
     name = gen_string("alpha")
-    image_path = '/var/lib/libvirt/images/{0}.img'.format(name)
 
     def _pxe_boot_host(self, mac):
         """PXE boot a unknown host"""
-        libvirt_server = 'qemu+tcp://{0}:16509/system'.format(
-            settings.server.hostname)
-        ssh.command('virt-install --hvm --network=bridge:virbr1, --mac={0} '
-                    '--pxe --name {1} --ram=1024 --vcpus=1 --os-type=linux '
-                    '--os-variant=rhel7 --disk path={2},size=10 --connect {3} '
-                    '--noautoconsole'
-                    .format(mac, self.name, self.image_path, libvirt_server))
-        sleep(30)
+        ssh.command(
+            u'virt-install --hvm --network=bridge:{0}, --mac={1} '
+            '--pxe --name {2} --ram=1024 --vcpus=1 --os-type=linux '
+            '--os-variant=rhel7 --disk path={3},size=10 --noautoconsole'
+            .format(settings.vlan_networking.bridge, mac, self.name,
+                    self.image_path),
+            hostname=self.libvirt_host
+        )
+        sleep(70)
```

Comment:
Yeah.. here we are discovering a unknown host and we don't know the IP in advance. So I put a sleep and we increased the timeout from `30 to `70` as we are using another server(external libvirt). Earlier we were using same server via nested virt.

I'm open for any suggestion to get rid of this sleep. thanks for review comments.